purchase_tier_validation/tests/test_tier_validation.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 ForgeFlow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo.tests import common
+from odoo.tests.common import tagged
+
+from .common import setup_test_model, teardown_test_model
+from .tier_validation_tester import TierValidationTester
+
+
+@tagged("post_install", "-at_install")
+class TestPurchaseTierValidation(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestPurchaseTierValidation, cls).setUpClass()
```

Comment:
can you set `tracking_disable` on the env?

Reply:
Updated, Thanks

Reply:
it won't work like that. You must override it in setUpClass like
```
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```